index.html

```diff
@@ -71,36 +73,41 @@ <h1 class="loading">选课平台
                     <span class="d d-3">.</span>
                 </h1>
                 <p>选课活动开始，请在选课前仔细阅读此说明。</p>
+                <noscript>
```

Comment:
Is that your idea to show a warning here?

Reply:
It will show when browser disable JavaScript, in case some users can't see the alert.